src/engine/specification.md

```diff
@@ -197,25 +223,35 @@ This structure contains the attributes required to initiate a payload build proc
 #### Response
 
 * result: `object`
-    - `status`: `enum` - `"SUCCESS" | "SYNCING"`
+    - `status`: `enum` - `"VALID" | "INVALID" | "SYNCING" | "ACCEPTED"`
+    - `latestValidHash`: `DATA|null`, 32 Bytes - the hash of the most recent *valid* block in the branch defined by payload and its ancestors
+    - `validationError`: `String|null` - a message providing additional details on the validation error if the payload is deemed `INVALID`
     - `payloadId`: `DATA|null`, 8 Bytes - identifier of the payload build process or `null`
-* error: code and message set in case an exception happens while updating the forkchoice or initiating the payload build process.
+* error: code and message set in case an exception happens while validating payload, updating the forkchoice or initiating the payload build process.
 
 #### Specification
 
-1. The values `(forkchoiceState.headBlockHash, forkchoiceState.finalizedBlockHash)` of this method call map on the `POS_FORKCHOICE_UPDATED` event of [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675#specification) and **MUST** be processed according to the specification defined in the EIP.
+1. Client software **SHOULD** initiate the sync process if `forkchoiceState.headBlockHash` references an unknown payload or a payload that can't be validated because requisite data is missing. The sync process is specified in the [Sync process](#sync-process) section. Additionally:
+    * Client software **MUST** return `{status: SYNCING, latestValidHash: null, validationError: null, payloadId: null}` if the sync process has been initiated.
```

Comment:
If the client doesn' follow the SHOULD here, MUST they still return SYNCING?

Reply:
`newBlock` says the following:

> Client software **SHOULD** initiate the sync process if requisite data for payload validation is missing. The sync process is specified in the [Sync process](#sync-process) section. Additionally:

The only difference is that it doesn't mentioned `headBlockHash` specifically. Should these actually be the same text? *And* if so, I would recommend hoisting this call into the `validationProcess`. `payloadId` can conditionally be returned as `null` if `SYNCING` toward the end of the list (outside `validationProcess`)

Reply:
> If the client doesn' follow the SHOULD here, MUST they still return SYNCING?

Yes, it must if we want to keep this semantics unchanged with respect to the previous version.

> And if so, I would recommend hoisting this call into the validationProcess.

Good suggestion. I was leaving it here only for explicitly specifying return values for each branch of method processing. But I think we can improve on return values as well by moving them in a separate section for this method, considering this move for `newPayload` as well -- will see how it applies.